### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by FlaPer87**
Wherever you used to do `import newpr` or `python newpr.y` you now need to do `import highfive.newpr` or `python highfive/newpr.py` respectively.

### CODE REVIEW

**Comment by nick29581** [comment]
(don't mind me, just testing something completely different)

File: highfive/newpr.py [hunk 1]
```diff
@@ -82,6 +83,12 @@ def send_then_quit(self, msg):
         self.quit()
 
 
+def _load_json_file(name):
```
Comment (nick29581) [inline highfive/newpr.py]: why does this have a `_` prefix?

File: highfive/newpr.py [hunk 2]
```diff
@@ -195,15 +202,13 @@ def choose_reviewer(repo, owner, diff, exclude):
         return 'test_user_selection_ignore_this'
 
     # Get JSON data on reviewers.
-    with open(repo + '.json') as rf:
-        reviewers = json.load(rf)
+    reviewers = json.loads(_load_json_file(repo + '.json'))
```
Comment (nick29581) [inline highfive/newpr.py]: could we move the `json.loads` call to `load_json_file` too?

File: highfive/newpr.py (hunk 1 above)
Comment (FlaPer87) [inline highfive/newpr.py]: that's the python convention to say something is private

File: highfive/newpr.py (hunk 2 above)
Comment (FlaPer87) [inline highfive/newpr.py]: yup. FWIW, the function was originally meant to load more than just json. I'm planning to refactor most of this code soon. I'll move it, though.

File: highfive/newpr.py (hunk 1 above)
Comment (nick29581) [inline highfive/newpr.py]: Yikes, Python

**Comment by nick29581** [comment]
Thanks a lot for this, looks good! Could you add a readme file or something with instructions for how to install highfive (I assume it needs a setup step now?) and how to run tests please?

**Comment by FlaPer87** [comment]
@nick29581 done :)

**Pull request merged**